libs/NfcCoreLib/lib/Nci/phNciNfc.h

```diff
@@ -287,6 +287,24 @@ typedef enum phNciNfc_NfceeModes
    PH_NCINFC_NFCEEDISC_UNKNOWN
 }phNciNfc_NfceeModes_t;
 
+/**
+* \ingroup grp_nci_nfc
+* \brief NFCEE Power and Link Control Info contains power related identifier..
+*/
+typedef enum phNciNfc_PowerLinkMode
+{
+    /** NFCEE  NFCC decides identifier*/
+    phNciNfc_NfceeNfccDecides = 0x00,
```

Comment:
My name recommendations:
- phNciNfc_PLM_NfccDecides
- phNciNfc_PLM_PowerSupplyAlwaysOn
- phNciNfc_PLM_ComLinkActiveWhenPowerOn
- phNciNfc_PLM_PowerAndComLinkAlwaysOn

Reply:
I have done that change for phNciNfc_PowerLinkModes_t but should this be changed also for phLibNfc_PowerLinkModes_t or the current name here are fine ?

Both enumeration are there because it is not possible to access phNciNfc_PowerLinkModes_t from LibNfc files.

Reply:
Having two enumerations is fine. The (sub-)names should also be used in phLibNfc_PowerLinkModes_t as well. Thanks.

Reply:
Just updated the pull request for phLibNfc_PowerLinkModes_t.